clarity/src/vm/analysis/type_checker/v2_1/natives/mod.rs

```diff
@@ -699,6 +700,48 @@ fn check_get_burn_block_info(
     )?)
 }
 
+fn check_get_stacks_block_info(
+    checker: &mut TypeChecker,
+    args: &[SymbolicExpression],
+    context: &TypingContext,
+) -> TypeResult {
+    check_arguments_at_least(2, args)?;
```

Comment:
I think you want `check_argument_count` here -- there should be exactly two arguments.

Reply:
Oops, copied from `get-block-info?` which I'm now remembering needed this for backwards compatibility.

Reply:
Fixed in 1fad8d226e8081b4204c6a6ecc44c5ff8038013b